modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -41,7 +41,18 @@ Write a short introductory paragraph that provides an overview of the module. Th
 NOTE: You can add text, tables, code examples, images, and other items to a step. However, these items must be connected to the step with a plus sign (+). Any items under the .Procedure heading and before one of the following approved headings that are not connected to the last step with a plus sign cannot be converted to DITA.
 
 ////
-Only the following block headings can be reliably mapped to DITA: Prerequisites, Procedure, Verification, Troubleshooting, Troubleshooting steps, Next steps, Next step, Additional resources. They must appear in this order and, with the exception of Additional resources, are only allowed in a procedure module. You can also use block headings in figure, table, and example titles.
+Only the following block headings can be reliably mapped to DITA:
+
+* Prerequisites or Prerequisite
+* Procedure
+* Verification, Results, or Result
+* Troubleshooting, Troubleshooting steps, or Troubleshooting step
+* Next steps or Next step
+* Additional resources
+
+With the exception of Additional resources, these titles are only allowed in a procedure module.
```

Comment:
Maybe clarify that only one of each item is allowed?  For example, Verification and Results together is not allowed

Reply:
In my experience, the clearer the limitations, the better